### PR TITLE
Create Hebrew translation

### PR DESCRIPTION
**Pull request by Eng2Heb**
Done

### CODE REVIEW

File: resources/language/Hebrew/strings.po [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+msgid "Songs"
+msgstr "פזמון"
+
+msgctxt "#30214"
+msgid "Shuffle Albums"
+msgstr "ערבב אלבומים"
+
+msgctxt "#30215"
+msgid "Thumbs up"
+msgstr "לייק"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po]: ו think אהבתי is better

File: resources/language/Hebrew/strings.po [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+
+msgctxt "#30216"
+msgid "Last added"
+msgstr "האחרון שנגמר"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po]: Should be נוספו אחרונים

File: resources/language/Hebrew/strings.po [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+msgid "Listen Now"
+msgstr "הקשב עכשיו"
+
+msgctxt "#30220"
+msgid "Top charts"
+msgstr "הכי גבוה במצעדים"
+
+msgctxt "#30221"
+msgid "New Releases"
+msgstr "שיחרורים חדשים"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po:228]: Maybe 'יצאו לאחרונה'?

File: resources/language/Hebrew/strings.po [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+# ###
+# Messages
+# ###
+msgctxt "#30101"
+msgid "Login failed"
+msgstr "הכניסה נכשלה"
+
+msgctxt "#30102"
+msgid "Invalid credentials or connectivity problems"
+msgstr "אישורי כניסה לא תקינים או בעיות קישור"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po]: I think 'אישורי כניסה' should be 'שם משתמש/סיסמה'

File: resources/language/Hebrew/strings.po [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+msgid "Fetching video IDs"
+msgstr "שליפת מזהי וידאו"
+
+msgctxt "#30409"
+msgid "Set Rating"
+msgstr "קבע דירוג"
+
+msgctxt "#30410"
+msgid "Thumbs up"
+msgstr "לייק"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po]: I think 'אהבתי' is better

File: resources/language/Hebrew/strings.po [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+
+msgctxt "#30411"
+msgid "Thumbs down"
+msgstr "לייק שלילי"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po]: 'לא אהבתי'

File: resources/language/Hebrew/strings.po [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+
+msgctxt "#30412"
+msgid "No thumbs"
+msgstr "אין לייק"
```
Comment (dvirtz) [inline resources/language/Hebrew/strings.po]: Maybe 'לא דורג'?

**Review by dvirtz — COMMENTED** [review]
Added some suggestions

File: resources/language/Hebrew/strings.po (hunk 1 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: Its better.

File: resources/language/Hebrew/strings.po (hunk 2 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: OK

File: resources/language/Hebrew/strings.po (hunk 3 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po:228]: Nop

File: resources/language/Hebrew/strings.po (hunk 4 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: OK

Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: OK

File: resources/language/Hebrew/strings.po (hunk 5 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: OK

File: resources/language/Hebrew/strings.po (hunk 6 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: OK

File: resources/language/Hebrew/strings.po (hunk 7 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po]: I'll think it over

**Comment by foreverguest** [comment]
Ok to merge?

**Comment by Eng2Heb** [comment]
Yes. I've made the necessary changes by Dvir's suggestions.

בתאריך 25 בספט׳ 2017 05:23 PM,‏ "Leonardo Cervo" <notifications@github.com>
כתב:

**Pull request merged**